tracing/src/subscriber.rs

```diff
@@ -38,4 +41,24 @@ where
     crate::dispatcher::set_global_default(crate::Dispatch::new(subscriber))
 }
 
+/// Sets the subscriber as the default for the duration of the lifetime of the
+/// returned [`DefaultGuard`]
+///
+/// The default subscriber is used when creating a new [`Span`] or
+/// [`Event`], _if no span is currently executing_. If a span is currently
+/// executing, new spans or events are dispatched to the subscriber that
+/// tagged that span, instead.
+///
+/// [`Span`]: ../span/struct.Span.html
+/// [`Subscriber`]: ../subscriber/trait.Subscriber.html
+/// [`Event`]: :../event/struct.Event.html
+/// [`DefaultGuard`]: ../../tracing-core/src/dispatcher.DefaultGuard.html
```

Comment:
This link URL is incorrect — that path won't exist in the generated RustDoc. I think this should be:
```suggestion
/// [`DefaultGuard`]: ../dispatcher/struct.DefaultGuard.html
```